src/main/java/hellos/Hellos.java

```diff
@@ -36,6 +37,11 @@ private String kkSaysHello() {
     return "KK says 'Hello!'\n";
   }
 
+
+  private String thomasSaysHello() {
+    return "Thomas says 'Hola!'\n";
+  }
+
```

Comment:
This both follows the format of the other greetings and passes the tests. It should be ready for use!